provider.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  backend "s3" {
+    bucket                  = "consultfstate"
+    region                  = "eu-central-1"
+    key                     = "terraform.tfstate"
+    shared_credentials_file = "./credentials"
```

Comment:
Don't use explicit credentials defining. It's an optional parameter and your keys should be stored in the default for terraformi location.  Your code in the future will be used by different members so they shouldn't specify anything custom.